src/pages/Login/styles.scss

```diff
@@ -0,0 +1,107 @@
+@import 'styles/styles.scss';
+
+.login-form {
+  background-color: $color-light-grey;
+  color: $color-black;
+  height: 100vh;
+  width: 50vw;
+
+  .login-msg {
+    font-size: $font-size;
+    margin-bottom: rem(50px);
+    margin-top: rem(100px);
+    max-width: rem(321px);
+    text-transform: none;
+    h1 {
+      font-size: $font-size-title;
+      font-weight: bold;
+      line-height: 23.44px;
+      margin-top: rem(100px);
+    }
+    h2 {
+      font-size: $font-size-subtitle;
+      font-weight: bold;
+    }
+  }
+
+  form {
+    width: rem(185px);
+
+    label {
+      font-size: $font-size-2;
+      font-weight: 600;
+      line-height: 13px;
+      margin-bottom: rem(5px);
+      text-transform: uppercase;
+    }
+
+    Input {
+      background-color: transparent;
+      font-size: $font-size-1;
+      margin-bottom: rem(17px);
+      width: rem(185px);
+    }
+
+    .error-message {
+      color: $color-red;
+      display: block;
+      margin-top: rem(-10rem);
+      margin-bottom: rem(15px);
+      font-size: $font-size-error;
+      text-transform: none;
+    }
+
+    .button-container {
+      align-items: center;
+      display: flex;
+      flex-direction: column;
+      width: rem(185px);
+
+      .button {
+        text-transform: uppercase;
+        font-weight: 600;
+        font-size: $font-size-2;
+        line-height: 12.89px;
+      }
+
+      a {
+        margin-top: 0.8rem;
+        text-decoration: none;
+        &.forgot-password {
+          color: $color-black;
+          font-size: $font-size-2;
+          font-weight: 600;
+          line-height: 100%;
+          margin-bottom: rem(50px);
```

Comment:
i would prefer if you use rem directly, this could help you https://nekocalc.com/px-to-rem-converter